aiida/restapi/translator/computer.py

```diff
@@ -75,9 +75,9 @@ def get_projectable_properties(self):
                 'type': 'int',
                 'is_display': False
             },
-            'name': {
-                'display_name': 'Name',
-                'help_text': 'Name of the object',
+            'label': {
+                'display_name': 'Label',
```

Comment:
to be fully consistent, one might argue that `display_name` should become `display_label`.
but I'm ok with keeping this inconsistency

Reply:
> I notice that the file `tests/static/graphs/graph1.aiida` was deleted - was this just no longer needed?

It wasn't deleted, it was just migrated since it needed to also perform the `name` -> `label` change in the archive. Due to the new archive format that is already there, the size was just reduced.